### ISSUE TITLE
problem building app

### ISSUE THREAD

**Issue by MirkoUgoliniDev** [moutend/gwca#5]
Hello ,

What version of Go are you using (go version)?
![image](https://user-images.githubusercontent.com/12691549/92475614-f7761980-f1dd-11ea-896e-fb0eef314c41.png)

What operating system and processor architecture are you using (go env)?
![image](https://user-images.githubusercontent.com/12691549/92475672-11176100-f1de-11ea-9364-3ba38e50f345.png)

What did you do?
I'm using   go-wca  inside persona project  

I use Visual studio code as IDE

but  in  the building phase   **go build -o bin/hello**   I got:

What did you expect to see?
app build without problem

What did you see instead?
go: finding module for package github.com/moutend/go-wca
..\..\pkg\mod\github.com\talkkonnect\volume-go@v0.0.0-20200223011643-173f1ce653ed\volume_windows.go:8:2: module github.com/moutend/go-wca@latest found (v0.2.0), but does not contain package github.com/moutend/go-wca

thanks in advance

**Comment by janekjan** [comment]
I think you are trying to `import "github.com/moutend/go-wca"` instead of `import "github.com/moutend/go-wca/pkg/wca"`.
I guess you have solved the problem after such a long time, but I hope somebody finds it useful.

**Comment by emmaly** [comment]
> I think you are trying to `import "github.com/moutend/go-wca"` instead of `import "github.com/moutend/go-wca/pkg/wca"`. I guess you have solved the problem after such a long time, but I hope somebody finds it useful.

This is absolutely right.  @MirkoUgoliniDev if this issue is no longer outstanding, I recommend closing this issue, or updating it to reflect current state if it does persist.  ❤️
